apps/language_server/lib/language_server/server.ex

```diff
@@ -391,6 +403,7 @@ defmodule ElixirLS.LanguageServer.Server do
       "hoverProvider" => true,
       "completionProvider" => %{"triggerCharacters" => ["."]},
       "definitionProvider" => true,
+      "referencesProvider" => true,
```

Comment:
The `Mix.Tasks.Xref.calls/0` function was added in 1.6, so we should add `References.supported?/0` function we call here like in the `Formatting` provider.